app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
```

Comment:
It seems like you are trying to implement a file copy function. However, you are not checking if the `cp` input command is correct. The user could input any command and it would still try to execute it. This could lead to unexpected behavior or errors. You should add a check to make sure that the command starts with `cp`.